applications/prompt-proto-service-lsstcomcam/values-usdfprod-prompt-processing.yaml

```diff
@@ -35,27 +33,37 @@ prompt-proto-service:
           pipelines: []
         - survey: BLOCK-T216  # twilight flats
           pipelines: []
-        - survey: BLOCK-T219  # pretty pictures
-          pipeliens: []
         - survey: BLOCK-T246  # instrument checkout
           pipelines: ['${PROMPT_PROCESSING_DIR}/pipelines/LSSTComCam/Isr-cal.yaml']
         - survey: BLOCK-T249  # AOS alignment
           pipelines: []
         - survey: BLOCK-T250  # TMA daytime checkout
           pipelines: []
+        - {survey: BLOCK-T306, pipelines: []}  # Manual observations during SV block
+        - {survey: BLOCK-T307, pipelines: []}  # Manual observations during SV block
+        - {survey: BLOCK-T308, pipelines: []}  # Manual observations during SV block
+        - {survey: BLOCK-T309, pipelines: []}  # Manual observations during SV block
+        - {survey: BLOCK-T310, pipelines: []}  # Manual observations during SV block
+        - {survey: BLOCK-T311, pipelines: []}  # Manual observations during SV block
+        - {survey: BLOCK-T312, pipelines: []}  # Manual observations during SV block
```

Comment:
Do we want to add `BLOCK-T318`, just because it might stay around for a bit longer?

Reply:
Do you think it might? It looked like one of the usual "run it a few times until we're satisfied" type tests.

Reply:
I don't know, and indeed it looks like that type you described.  Either including it here or not is fine to me.

Reply:
Then I'll leave it out for now (though I guess we're running out of "for now"!).